test/suites/apply/fixtures/cluster_file_for_test.yaml

```diff
@@ -15,33 +15,32 @@
 apiVersion: zlink.aliyun.com/v1alpha1
 kind: Cluster
 metadata:
+  creationTimestamp: null
   name: my-test-cluster
 spec:
-  image: sealer-io/kubernetes:v1.19.8
-  provider: ALI_CLOUD
-  ssh:
-    passwd: Sealer123
-    pk: xxx
-    pkPasswd: xxx
-    user: root
-  network:
-    podCIDR: 100.64.0.0/10
-    svcCIDR: 10.96.0.0/22
   certSANS:
-    - aliyun-inc.com
-    - 10.0.0.2
-
+  - aliyun-inc.com
+  - 10.0.0.2
+  image: registry.cn-qingdao.aliyuncs.com/sealer-io/kubernetes:v1.22.4-test
   masters:
-    cpu: 4
-    memory: 4
-    count: 1
-    systemDisk: 100
+    count: "1"
+    cpu: "2"
     dataDisks:
-      - 100
+    - "100"
+    memory: "2"
+    systemDisk: "100"
+  network:
+    podCIDR: 100.64.0.0/10
+    svcCIDR: 10.96.0.0/22
   nodes:
-    cpu: 4
-    memory: 4
-    count: 1
-    systemDisk: 100
+    count: "1"
+    cpu: "2"
     dataDisks:
-      - 100
+    - "100"
+    memory: "2"
+    systemDisk: "100"
+  provider: CONTAINER
```

Comment:
do we still need v1.cluster struct to launch infra ? Or just prepare container node as test request ?